policy/policy.go

```diff
@@ -24,23 +24,28 @@ import (
 	"github.com/testifysec/go-witness/cryptoutil"
 	"github.com/testifysec/go-witness/log"
 	"github.com/testifysec/go-witness/source"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const PolicyPredicate = "https://witness.testifysec.com/policy/v0.1"
 
+// +kubebuilder:object:generate=true
 type Policy struct {
-	Expires              time.Time            `json:"expires"`
+	Expires              metav1.Time          `json:"expires"`
```

Comment:
Does this affect the json marshalling/unmarshalling of this struct at all?

Reply:
Possibly? If it does, for the better I think. But, we probably won't notice. From the Kubebuilder book:
> There’s one other special type that we use: metav1.Time. This functions identically to time.Time, except that it has a fixed, portable serialization format.